docker-compose.yml

```diff
@@ -34,3 +44,11 @@ elasticsearch:
 
 redis:
   image: redis
+
+rabbitmq:
+  image: rabbitmq:3-management
+  hostname: olympia
+  name: olympia
+  ports:
+    - "5672:5672"
```

Comment:
This should probably be `expose` rather than ports. Exposing ports outside of the container set might cause problems for people already running these services.